cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -69,6 +69,7 @@ spec:
             - --exempt-namespace={{ .Release.Namespace }}
             - --operation=webhook
             - --enable-mutation={{ .Values.experimentalEnableMutation}}
+            - --disable-opa-builtin={{ .Values.builtinHttpSend }}
```

Comment:
What happens here if no value is provided? Will this still work?

Reply:
It works with unset value in this case, but seems like a good idea to remove empty string in FlagSet.ToSlice().